core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala

```diff
@@ -1348,6 +1368,22 @@ private[spark] object SparkSubmitUtils {
     }
   }
 
+  private[deploy] def getSubmitOperations(master: String): SparkSubmitOperation = {
+    val loader = Utils.getContextOrSparkClassLoader
+    val serviceLoaders =
+      ServiceLoader.load(classOf[SparkSubmitOperation], loader)
+        .asScala
+        .filter(_.supports(master))
+
+    serviceLoaders.size match {
+      case x if x>1 =>
+        throw new SparkException(s"Multiple($x) external SparkSubmitOperations " +
+          s"clients registered for Master url ${master}.")
+      case 1 => serviceLoaders.headOption.get
+      case _ => throw new IllegalArgumentException(s"No external SparkSubmitOperations " +
```

Comment:
Same.

Also, start mutli-line statements on their own line (like in the previous case).